components/profile_popover.jsx

```diff
@@ -194,6 +194,7 @@ class ProfilePopover extends React.Component {
         delete popoverProps.hide;
         delete popoverProps.isRHS;
         delete popoverProps.hasMention;
+        delete popoverProps.dispatch;
```

Comment:
This is unrelated to the bug but I was in the area and it was causing a bunch of JS warnings.